tools/cabana/mainwin.cc

```diff
@@ -135,6 +138,10 @@ void MainWindow::createActions() {
   commands_act->setDefaultWidget(new QUndoView(UndoStack::instance()));
   commands_menu->addAction(commands_act);
 
+  edit_menu->addSeparator();
+  edit_menu->addAction(tr("Reset Window Layout"),
+                       [=]() { this->restoreState(this->default_state); });
```

Comment:
how about change it to `[this]() { restoreState(default_state); });` ?
we don't use `this` pointer explicitly If there is no particular reason.

Reply:
@deanlee Done, thanks! I think I had a weird feeling "oh, I'm in a lambda scope so I should be really explicit about what I'm binding". But it's not necessary at all!